src/ray/gcs/gcs_server/gcs_health_check_manager.h

```diff
@@ -82,6 +82,12 @@ class GcsHealthCheckManager {
   /// \return A list of node id which are being monitored by this class.
   std::vector<NodeID> GetAllNodes() const;
 
+  /// Mark the given node as healthy, so health check manager could save some checking
+  /// rpcs. Notice: have to invoke from io-context thread.
```

Comment:
I don't think this comment is accurate

Reply:
I think it's describing what it does? Any detailed comment on which part inaccurate?

Reply:
Fixed.